src/Symfony/Component/PropertyInfo/Tests/PropertyInfoCacheExtractorTest.php

```diff
@@ -76,4 +82,88 @@ public function testIsInitializable()
         parent::testIsInitializable();
         parent::testIsInitializable();
     }
+
+    /**
+     * @dataProvider provideNestedExtractourWithoutGetTypeImplementationData
+     */
+    public function testNestedExtractorWithoutGetTypeImplementation(string $property, ?Type $expectedType)
```

Comment:
this test should be marked as `@group legacy` as it covers a BC layer (and so should be removed once we upgrade to 8.0)